From 0e571ccff4783adaa29da73d60c96d762fe6af58 Mon Sep 17 00:00:00 2001
From: Zhao Qiang <qiang.zhao@nxp.com>
Date: Mon, 18 Jan 2016 16:09:49 +0800
Subject: [PATCH 0083/1383] T1040RDB: add tdm riser card node to t1040rdb

add tdm riser card node to t1040rdb.dts

Signed-off-by: Zhao Qiang <qiang.zhao@nxp.com>
[Original patch taken from QorIQ-SDK-V2.0-20160527-yocto]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 arch/powerpc/boot/dts/fsl/t1040rdb.dts | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/powerpc/boot/dts/fsl/t1040rdb.dts b/arch/powerpc/boot/dts/fsl/t1040rdb.dts
index 8f9e65b..4a3371d 100644
--- a/arch/powerpc/boot/dts/fsl/t1040rdb.dts
+++ b/arch/powerpc/boot/dts/fsl/t1040rdb.dts
@@ -43,6 +43,16 @@
 			compatible = "fsl,t1040rdb-cpld";
 		};
 	};
+
+	soc: soc@ffe000000 {
+		spi@110000 {
+			slic@3 {
+				compatible = "maxim,ds26522";
+				reg = <3>;
+				spi-max-frequency = <2000000>; /* input clock */
+			};
+		};
+	};
 };
 
 /include/ "t1040si-post.dtsi"
-- 
2.8.1

